### PR TITLE
Paul/docs

### PR DESCRIPTION
**Pull request by paulcjh**

### CODE REVIEW

File: .github/workflows/publish-docs.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: Publish Pipeline-ai 📖 docs to readme.com
+on:
+  push:
+    branches: 
+      - paul/docs
```
Comment (bjornaer) [inline .github/workflows/publish-docs.yml:5]: I guess you might want to change target branch?

File: .gitignore [hunk 2]
```diff
@@ -135,3 +135,4 @@ examples/examples
 local/
 pipeline-bak/
 *.env
+.DS_Store
```
Comment (bjornaer) [inline .gitignore:138]: Good man!

**Review by bjornaer — CHANGES_REQUESTED** [review]
could you pull main branch to sort out any conflicts if there are any?

**Comment by bjornaer** [comment]
> @bjornaer This is a draft branch atm

did not notice, I got pinged by github and reviewed haha nw

**Comment by paulcjh** [comment]
> > @bjornaer This is a draft branch atm
> 
> did not notice, I got pinged by github and reviewed haha nw

Nw!

**Pull request closed without merging**